apps/condo/lang/en.json

```diff
@@ -181,6 +181,9 @@
   "pages.condo.property.form.EmptyResidents.text": "You can upload excel file to import residents",
   "pages.condo.property.form.field.Name": "Property name",
   "pages.condo.property.form.ResidentsTabTitle": "Residents",
+  "pages.condo.property.form.DeleteLabel": "Delete property",
+  "pages.condo.property.form.ConfirmDeleteTitle": "Delete property?",
+  "pages.condo.property.form.ConfirmDeleteMessage": "All connected to property data will be lost",
```

Comment:
I would name it:

All data, related to this property, will be lost

I think **connected** is harder to understand to non-IT people

Reply:
Done